src/EnergyPlus/DaylightingManager.cc

```diff
@@ -7341,76 +7327,76 @@ void DayltgElecLightingControl(EnergyPlusData &state)
             continue;
         }
 
+        // Electric lighting power reduction factor for a zone due to daylighting
+        Real64 &TotReduction = thisDaylightControl.PowerReductionFactor;
```

Comment:
This comment is outdated. It should be " . . . for a given daylighting control." This could apply to a space or zone presently, with the future goal of applying to specific Lights objects within a space or zone.

Reply:
Addressed in cbf31da66ff2806fb20d91777273adbac77f399d